velox/docs/develop/types.rst

```diff
@@ -174,6 +174,14 @@ UUID                      HUGEINT
 IPADDRESS                 HUGEINT
 IPPREFIX                  ROW(HUGEINT,TINYINT)
 GEOMETRY                  VARBINARY
+TDIGEST(DOUBLE)           VARBINARY
```

Comment:
What does TDIGEST(DOUBLE) denote ? Especially since they all map to VARBINARY.

Also Q(T)DIGEST(DOUBLE) is repeated in the list.

Reply:
Thank you I updated this. 

TDigest and QDigest are types, that only work with a parametrized type. TDigest accepts double, whereas, QDigest works with bigint, real or double

E.g. cast('xyz' as tdigest) will not work as "tdigest" is not a valid type
cast('xyz' as tdigest(double)) would be the way to use T-Digest

Reply:
@natashasehgal I suggest to list only TDIGEST and QDIGEST in the table and explain that they are parameterized in the text below. This would be consistent with how we document DECIMAL type.

Reply:
@natashasehgal Would you take a look at this suggestion?